javascript/src/functions.js

```diff
@@ -1,32 +1,41 @@
-const Game = require('./game.js');
+const Game = require("./game.js");
 
-function play(seed) {
+function doRandomRollNumber() {
+  return Math.floor(Math.random() * 6) + 1;
+}
 
-    if (seed != undefined) {
-        require('seedrandom')(seed, { global: true });
-    }
+function wouldAnswerIncorrectly() {
```

Comment:
For this and the above `doRandomRollNumber`, I wanted to document what each random number was actually doing. Not sure I am completely happy with this, but it did allow me to export these, so maybe future tests could include these functions.